src/CodeGen_LLVM.cpp

```diff
@@ -2978,19 +3018,19 @@ Constant *CodeGen_LLVM::create_string_constant(const string &s) {
         data.reserve(s.size()+1);
         data.insert(data.end(), s.begin(), s.end());
         data.push_back(0);
-        Constant *val = create_constant_binary_blob(data, "str");
+        Constant *val = create_binary_blob(data, "str");
         string_constants[s] = val;
         return val;
     } else {
         return iter->second;
     }
 }
 
-Constant *CodeGen_LLVM::create_constant_binary_blob(const vector<char> &data, const string &name) {
-
+Constant *CodeGen_LLVM::create_binary_blob(const vector<char> &data, const string &name) {
     llvm::Type *type = ArrayType::get(i8, data.size());
+    bool constant = data.size() > 1024;
```

Comment:
This is a hacky way to determine if the data is constant or not. I will fix this.